TPKeyboardAvoiding/UIScrollView+TPKeyboardAvoidingAdditions.m

```diff
@@ -42,6 +42,9 @@ - (TPKeyboardAvoidingState*)keyboardAvoidingState {
 }
 
 - (void)TPKeyboardAvoiding_keyboardWillShow:(NSNotification*)notification {
+    if (![[NSProcessInfo processInfo] respondsToSelector:@selector(isOperatingSystemAtLeastVersion:)]){
```

Comment:
Comment pleaseee (this isn't immediately clear as an iOS check for those unfamiliar with `isOperatingSystemAtLeastVersion`)